classes/02class/exercises/c02-network07/starkmatt/c02-network07.txt

```diff
@@ -23,9 +23,11 @@ Those are having a one way connection out but not back in, you're able to use an
 
 Can you delete the destination route to your VPC network? Why?
 - The local VPC that was created initally ?
-- It would be unwise, because you won't be able to access the Environment.
+- You cannot delete any VPC that was created by default (AWS Does not allow it)
```

Comment:
That is not right, you can delete the default VPC's and it is actually a good practice in big enterprise companies if you are not going to use a specific region

Reply:
You're right!
Default VPC is created for customers by AWS on 172.31.0.0/16.
They state that _"A default VPC is suitable for getting started quickly, and for launching public instances such as a blog or simple website"_
If there is already a default VPC in the region, you cannot create another one!

Reply:
Can you update the file with this information @starkmatt ?